src/neptune/common/deprecation.py

```diff
@@ -44,11 +53,22 @@ def get_user_code_stack_level():
     return 2
 
 
-def warn_once(message: str, *, exception: type(Exception) = NeptuneDeprecationWarning):
+def warn_once(message: str, *, exception: type(Exception) = None):
+    if None:
+        exception = NeptuneDeprecationWarning
+
     if message not in warned_once:
         warnings.warn(
             message=message,
             category=exception,
             stacklevel=get_user_code_stack_level(),
         )
         warned_once.add(message)
+
+
+def warn_about_unsupported_type(type_str: str):
+    warn_once(
+        message=f"""The type of the object you're logging is not supported by Neptune ({type_str}).
```

Comment:
@normandy7, that's for you. Remember that warnings should be more compact than exceptions :wink:

Reply:
So you're you're saying the current message should be condensed? I can even shorten the docs URL :)

**Q:** If it's no longer a named exception, will the user see `NeptuneUnsupportedType` in the output? (Asking for docs search purposes)

Reply:
> So you're you're saying the current message should be condensed? 
Not really. Could be even longer. I was referencing that in case of exceptions there are some "fancy" things like newlines, separators etc.

Reply:
>Q: If it's no longer a named exception, will the user see NeptuneUnsupportedType in the output? (Asking for docs search purposes)

Currently it looks like this:
```
>>> class Obj:
...     pass
...
>>> run['a'] = Obj()
<stdin>:1: NeptuneUnsupportedType: The type of the object you're logging is not supported by Neptune (<class '__main__.Obj'>).
        Need help? -> https://docs.neptune.ai/help/value_of_unsupported_type
```